### DISCUSSION TITLE
Blank screens, why?

### DISCUSSION THREAD

**Discussion by Baantjer** [winsiderss/systeminformer#1390] [Questions / Answers] [answered]
I am presently using Process hacker V. 3.0.4953 And both of the screens which are meant to show information on the disk and firewall are completely blank. The other screens show the information with no issues.
I can see only view the column headers.
Why is this and how do I get them to display information? 
Screenshots below
![disk](https://user-images.githubusercontent.com/114300941/192106241-586db814-6833-42b4-91c7-0b3efdb64cd3.jpg)

![firewall](https://user-images.githubusercontent.com/114300941/192106259-d5992811-2724-4b1d-b489-4d5c3e530fae.jpg)

**Comment by dmex** [comment, marked as answer]
WIndows 7 doesn't have much support for these features. You'll need Windows 8 or above.

**Reply by Baantjer** [reply]
thank you for the explanation.

**Reply by dmex** [reply]
The Disk tab should work fine on Windows 7 but only one application can capture Disk events.
